workers/unity/Packages/io.improbable.worker.sdk.mobile/BuildPostProcessXCode.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using UnityEditor;
+using UnityEditor.Callbacks;
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile
+{
+    public static class BuildPostProcessXCode
+    {
+        private static Type pbxType;
+
+        [PostProcessBuild]
+        public static void OnPostProcessBuild(BuildTarget buildTarget, string path)
+        {
+            if (buildTarget != BuildTarget.iOS)
+            {
+                return;
+            }
+
+            Debug.Log("PostProcessBuild");
+
+            var assemblies = AppDomain.CurrentDomain.GetAssemblies();
+            foreach (var assembly in assemblies)
+            {
+                var type = assembly.GetTypes().FirstOrDefault(t => t.FullName == "UnityEditor.iOS.Xcode.PBXProject");
+                if (type != null)
+                {
+                    pbxType = type;
+                    break;
+                }
+            }
+
+            //pbxType = Type.GetType("UnityEditor.iOS.Xcode.PBXProject");
+            if (pbxType == null)
+            {
+                Debug.LogWarning("iOS Support not installed.");
```

Comment:
maybe also tell them what to do when that happens

Reply:
Replaced this with an exception as this should only show up when Unity changes their API. The exception will make sure we catch it in buildkite.